app/controllers/admin/booths_controller.rb

```diff
@@ -49,7 +51,12 @@ def destroy
     end
 
     def accept
-      update_state(:accept, 'Booth accepted!')
+      if !maximum_accepted_booths?
```

Comment:
Instead of using negation in your if statement, why not switch the contents of if and else?
So that this reads
```
if maximum_accepted_booths?
  redirect ...
else
  update_state()
end
```